model/src/main/java/com/redhat/ipaas/model/WithTags.java

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (C) 2016 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.redhat.ipaas.model;
+
+import java.util.List;
+import java.util.Optional;
+
+public interface WithTags {
+
+    Optional<List<String>> getTags();
```

Comment:
Should this be a set rather than a list? Semantically doesn't allow duplicates so makes more sense to me.